test/testconstructors.cpp

```diff
@@ -3589,7 +3590,9 @@ class TestConstructors : public TestFixture {
 
     void uninitVarInheritClassInit() {
         Settings s;
-        s.libraries.emplace_back("vcl");
+        // TODO: test should probably not pass without library
```

Comment:
@danmar Please have a look at this. It was added in 03445c01c1b9db9c7f1abe7da31ea31fba6350dc.

Reply:
Loading a library with the name "vcl" triggers a bit of hard coded handling in cppcheck. That is unfortunate. But I think the code is intentional at least.

Reply:
Yes, but the test never triggers that code since it does pass without the library being loaded.

Reply:
hmm I believe I wanted to test that there would be no FP. I am confused why you could remove the vcl library and still don't get false positives.

Reply:
Please take a look. It doesn't have to be tackled within this PR though.